frontend/app/routes/protected/person-case/review.tsx

```diff
@@ -60,6 +66,14 @@ export async function action({ context, params, request }: Route.ActionArgs) {
     }
 
     case 'next': {
+      try {
+        const response = await sinApplicationService.submitSinApplication(sinApplicationRequest);
+        //TODO: store the response in session
+        console.log('SIN Application submitted successfully:', response);
+      } catch (err) {
+        console.log('Error submitting SIN application:', err);
+      }
```

Comment:
This code should be moved to the state machine's `entry` action for its final state. For now, you can continue to keep it here and we can look at moving it once the API we have to call stabilizes.